src/rpc/rpcquorums.cpp

```diff
@@ -54,7 +54,7 @@ UniValue quorum_list(const JSONRPCRequest& request)
 
     UniValue ret(UniValue::VOBJ);
 
-    for (auto& p : Params().GetConsensus().llmqs) {
+    for (auto& p : llmq::CLLMQUtils::GetEnabledQuorumTypes(chainActive.Tip())) {
```

Comment:
Would it not be the tip before this? (I actually think not, but want to through it out there)

Reply:
Basically we should ensure data consistency in different parts of code e.g. it's `chainActive.Tip()` in `CreateNewBlock` and `pindex->pprev` in `CQuorumBlockProcessor::ProcessBlock`. For RPCs it's kind of not _that_ critical but IMO it makes sense to use chain tip here to see the same data miners do.

Reply:
Okay, I think that makes sense